github/ipfs.yml

```diff
@@ -5256,9 +5239,7 @@ repositories:
         enforce_admins: false
         lock_branch: false
         push_restrictions:
-          - /aschmahmann
-          - /gmasgras
-          - /thattommyhall
+          - ipfs/kubo-maintainers
```

Comment:
@ipfs/ipdx how can I set this to push restrictions on, but with no associated group? IIUC there's nothing you can do anyway to stop admins from pushing (or maintainers from pushing with approval) so adding groups here seems unnecessary provided you can keep the restrictions enabled.